mopro-ffi/src/app_config/constants.rs

```diff
@@ -0,0 +1,171 @@
+pub const ARCH_X86_64: &str = "x86_64";
+pub const ARCH_ARM_64: &str = "aarch64";
+pub const ARCH_I686: &str = "x86";
+pub const ARCH_ARM_V7_ABI: &str = "armeabi-v7a";
+pub const ARCH_ARM_64_V8: &str = "arm64-v8a";
+
+pub const ENV_CONFIG: &str = "CONFIGURATION";
+pub const ENV_ANDROID_ARCHS: &str = "ANDROID_ARCHS";
+pub const ENV_IOS_ARCHS: &str = "IOS_ARCHS";
+
+#[derive(Debug, Copy, Clone, PartialEq, Eq)]
+pub enum Mode {
```

Comment:
moved `Mode` and `Arch` here due to module dependency. `mopro-ffi` should not depend on `CLI`.